RoverCore/RoverCore.Web/Views/Shared/Layouts/Back/_Footer.cshtml

```diff
@@ -1,4 +1,4 @@
-@using RoverCore.Web.Models
+@using RoverCore.Domain.Entities
 @inject ApplicationSettings Settings
```

Comment:
Combine these into one line that's fully qualified:
@inject RoverCore.Domain.Entities.ApplicationSettings Settings